src/intops/impl/inlineasm/arm64.nim

```diff
@@ -0,0 +1,62 @@
+## Arithmetic operations for integers implemented in ARM64 GCC/Clang inline Assembly.
+
+import ../../consts
+
+when cpuArm64 and compilerGccCompatible and canUseInlineAsm:
+  {.push inline, noinit.}
```

Comment:
I think this should be `{.push raises: [], inline, noinit, gcsafe.}` throughout.

Reply:
Done